fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -635,7 +673,8 @@ ACTOR Future<int> dumpData(Database cx,
                            Key rangeBegin,
                            PromiseStream<Future<Void>> addActor,
                            FlowLock* commitLock,
-                           Reference<KeyRangeMap<Version>> keyVersion) {
+                           Reference<KeyRangeMap<Version>> keyVersion,
```

Comment:
unrealted: can we please rename this method `dumpData` is confusing to follow the utility of this method :)

Reply:
Discussed with steve and we think `kvMutationLogToTransactions` is a good name